tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,96 @@
+
+    def test_todo(self):
+        text = ['a == b']
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': ()})
+            ]
+        }
+        with execute_bear(self.python_uut, "F", text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result, [])
+
+        text = ['# todo 123']
+        dep_results = {
+            'AnnotationBear': {}
+        }
+        with execute_bear(self.python_uut, "F", text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result[0].diffs, None)
+            self.assertEqual(result[0].affected_code[0].start.line, 1)
+            self.assertEqual(result[0].affected_code[0].start.column, 1)
+
+        text = ['# todo 123']
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type(123)
+            ]
+        }
+        with execute_bear(self.python_uut, "F", text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result[0].diffs, None)
+            self.assertEqual(result[0].affected_code[0].start.line, 1)
+            self.assertEqual(result[0].affected_code[0].start.column, 1)
+
+        text = ['# todo 123\n']
+        comments = (SourceRange.from_values("F", 1, 1),
+                    SourceRange.from_values("F", 1, 40))
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': comments})
+            ]
+        }
+        with execute_bear(self.python_uut, "F", text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result[0].diffs['F'].unified_diff,
+                             '--- \n'
+                             '+++ \n'
+                             '@@ -1 +0,0 @@\n'
+                             '-# todo 123\n')
+
+        text = ['test = 55 # todo 123\n']
+        comments = (SourceRange.from_values("F", 1, 11),
+                    SourceRange.from_values("F", 1, 23))
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': comments})
+            ]
+        }
+        with execute_bear(self.python_uut, "F", text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result[0].diffs['F'].unified_diff,
+                             '--- \n'
+                             '+++ \n'
+                             '@@ -1 +1 @@\n'
+                             '-test = 55 # todo 123\n'
+                             '+test = 55\n')
+
+    def test_todo_outside_of_comment(self):
+        section1 = Section("")
+        section1.append(Setting('language', 'python3'))
+        section1.append(Setting('keywords', 'todo'))
+        python_uut = KeywordBear(section1, Queue())
+
+        text = ['todo = 123\n']
```

Comment:
this seems to be reoccurring as well, can we store it in self and not define it multiple times?